libs/newsletter-workflow/src/lib/steps/launchNewsletter/isReady.ts

```diff
@@ -45,7 +45,7 @@ export const isReadyLayout: WizardStepLayout<LaunchService> = {
 
 		return populated;
 	},
-	label: 'check if ready',
+	label: 'Check if ready',
```

Comment:
The labels have all been created in lowercase to make them consistent with the ones that aren't specifically specified because the autogenerated version is fine as it is (usually the case when it's a single word)

Reply:
In any case, this step has been renamed in my WIP PR to add steps into the Launch wizard

Reply:
But then I hadn't spotted that somewhere along the line all the breadcrumbs have been changed so that we're no longer using defaults anywhere